infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -99,6 +99,34 @@ boolean checkBootstrap(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r)
 
     // Helper Methods
 
+    /**
+     * Handle a force layout request. Force layout enables the server to bypass consensus
+     * and accept a new layout.
+     *
+     * @param msg              corfu message containing LAYOUT_FORCE
+     * @param ctx              netty ChannelHandlerContext
+     * @param r                server router
+     * @param isMetricsEnabled True if metrics are enabled, False otherwise
+     */
+    @ServerHandler(type = CorfuMsgType.LAYOUT_FORCE)
+    public synchronized void handleLayoutForce(@Nonnull CorfuPayloadMsg<LayoutCommittedRequest> msg,
```

Comment:
can't you reuse handleCommitted message ?

Reply:
Done.